creatures/creatureAttr.cpp

```diff
@@ -516,9 +516,11 @@ void Creature::crtReset() {
     hp.setName("Hp");
     mp.setName("Mp");
 
-    auto* pThis = dynamic_cast<Player*>(this);
-    if(pThis) {
-        pThis->focus.setName("Focus");
+    if(isPlayer()) {
+        auto* pThis = dynamic_cast<Player*>(this);
```

Comment:
For now, comment out the broken lines with an explanation.  Not the ideal fix but I'll approve after that for now.